metrics/src/sender/postgres.cpp

```diff
@@ -71,6 +71,7 @@ auto pg_sender_t::on_send_timer(const std::error_code& ec) -> void {
 
 auto pg_sender_t::on_gc_timer(const std::error_code& ec) -> void {
     if(!ec) {
+        COCAINE_LOG_DEBUG(logger, "running gc for metrics");
```

Comment:
Either use capital letters for abbreviations or expand them: "GC" or "garbage collector".